content/en/news/releases/1.17.x/announcing-1.17/_index.md

```diff
@@ -0,0 +1,79 @@
+
+{{< relnote >}}
+
+{{< tip >}}
+Istio 1.17.0 is officially supported on Kubernetes versions `1.23` to `1.26`.
+{{< /tip >}}
+
+## What's new
+
+Since the 1.16 release we’ve added some important new features and marked some of our existing features as Beta signaling that they’re ready for production use. Here are some highlights:
+
+### Canary upgrade and revision tags were promoted to Beta
+
+Basic support for upgrading the service mesh following a canary pattern using revisions was introduced in the Istio 1.6 release. Using this approach, you can run multiple control planes side-by-side without impacting an existing deployment and slowly migrate workloads from the old control plane to the new. In Istio 1.10, revision tags was introduced as an improvement to canary upgrades to help reduce the number of changes an operator has to make to use revisions, and safely upgrade an Istio control plane. This is a very widely adopted and used feature by our users in production. All integration tests and end-to-end tests covering documentation have been completed for this feature to graduate to Beta.  
+
+### Helm installation was promoted to Beta
+
+Helm based installation of Istio, first introduced in Istio 0.4, has graduated to Beta. It is one of the most widely used methods to install Istio in production. All requirements to promote this feature to Beta were completed in this release including updating integration tests to use helm charts for install/upgrade, updating Helm integration tests and documenting advanced Helm chart customization and attributes in `values.yaml`.
+
+### Upgraded support for the Kubernetes Gateway API
+
+Istio's implementation of the [Gateway API](https://gateway-api.sigs.k8s.io/) has been moved to, and is now fully compliant with, the latest version of the API (0.6.1).
+
+### Enhancements to Istio dual-stack feature
+
+Added support  to generate configuration for Istio gateway with dual stack enabled. `IPv6` support in dual-stack mode was added in Kubernetes in version 1.16 and graduated to stable in the 1.22 release. Istio dual stack support is an alpha feature and was introduced in release 1.16. This is an area under [active development]( https://github.com/istio/istio/issues/40394).
```

Comment:
Hi @johnma14, there are some prepare changes for dual stack support in Istio 1.16 release, but it can not work at all. So I suggest that we can declare that the general dual stack support for Istio should begin with 1.17 release as an alpha feature.
Besides that I also added release notes for every change in this release. Thanks!

Reply:
@zhlsunshine thank you for your input. So for 1.17, when we say dual-stack works as an alpha feature, can we explicitly list what works and what does not. For example, if we can explicitly state something like this:

``` We can deploy a service on single or dual stack ip family on a dual stack cluster for 1.17. ```

Reply:
Hi @johnma14, Yeah, that would be great, and how about the explicitly state as below:
```
We can deploy a service with single or dual stack ip family on a dual stack cluster for 1.17. For instance, user can separately deploy 3 services with ipv4 only, ipv6 only and dual stack ip families on a dual stack k8s cluster, then these services can be accessible for between each other.
```